ui/src/app/components/help-sidebar.tsx

```diff
@@ -328,6 +333,25 @@ export const HelpSidebar = fp.flow(withCurrentWorkspace(), withUserProfile())(
       }
     }
 
+    sidebarContainerStyles(activeIcon, notebookStyles) {
```

Comment:
If there's a typescriptier or more concise way to do this, let me know. I gather there is a preference for immutables (and don't know the syntax for pushing all elements onto an existing object anyways)

Reply:
how about passing the parameter notebookStyles to the style object sidebarContainer itself and return the height and top based on the parameter ? I think i might  be able to find an example in our code somewhere

Reply:
Had an offline discussion, we do not have any examples now